lib/spack/spack/util/gpg.py

```diff
@@ -13,7 +13,7 @@
 
 _gnupg_version_re = r"^gpg \(GnuPG\) (.*)$"
 
-GNUPGHOME = spack.paths.gpg_path
+GNUPGHOME = os.getenv('SPACK_GNUPGHOME', spack.paths.gpg_path)
```

Comment:
I'm just wondering: since gnupg looks for the `GNUPGHOME` environment variable, should we use that here instead of `SPACK_GNUPGHOME`?

Reply:
GNUPGHOME is recognized by gpg out of the box, and so I can imagine a situation where a user sets it in one of their rc files, just for normal usage, but wouldn’t necessarily want it to be picked up by Spack. By prefixing with SPACK_, you can thus maintain your custom GNUPGHOME setting for gpg work outside of Spack, while still being able to customize the Spack GPG dir. If you do want them to be the same, you can still achieve that here, but this way you have the option of keeping them separate.